src/guide/bash.md

```diff
@@ -2,20 +2,27 @@
 
 ## 0. A brief primer on CLI applications
 
-This is as good a place as any to discuss what `iroha_client_cli` is and what you should expect it to be able to do. Most users think that everything that's run inside the terminal is a CLI program. This is not the case. A **Command-line Interface** is a glorified scripting language, that you interact with using the shell. These programs are run multiple times, and given different _arguments_: `--flag1` `file2` `--help`, etc, depending on what you want to do. A single session doesn't begin with you opening the program, and end with closing it. When you've stopped interacting the `iroha_client_cli` the session is finished.
+This is as good a place as any to discuss what `iroha_client_cli` is
+and what you should expect it to be able to do. Most users think that
+everything that's run inside the terminal is a CLI program. This is
```

Comment:
Actually do you think we should do it everywhere?

Reply:
Some markdown engines interpret line breaks as line breaks, so the layout in the page may be sub-optimal. Some other markdown engines ignore line breaks unless there's an empty line.

Reply:
So if you have it everywhere the PRs are much easier to review, but it could mess up the formatting.

Reply:
Thank you for following up in DM about the matter. Will perform my line breaks more carefully.